documentation/modules/exploit/winrm/winrm_script_exec.md

```diff
@@ -0,0 +1,146 @@
+## Vulnerable Application
+
+WinRM, is a Windows-native built-in remote management protocol in its simplest form that uses Simple Object Access Protocol to interface with remote computers and servers, as well as Operating Systems and applications. It handles remote connections by means of the WS-Management Protocol, which is based on SOAP (Simple Object Access Protocol). 
+This module uses valid credentials to login to the WinRM service and execute a payload. It has two available methods for payload delivery: Powershell 2.0 and VBS CmdStager. This module will check if Poweshell 2.0 is available, and if so then it will use that method. Otherwise it falls back to the VBS CmdStager which is less stealthy.
+
+**IMPORTANT:-** If targetting an x64 system with the Poweshell method, one must select an x64 payload. An x86 payload will never return. 
+
+## Example Usage
+
+## Windows 2008
```

Comment:
One more '#' as in it should look like: -

+### Example Usage
+
+### Windows 2008